models/marts/dim_customers.sql

```diff
@@ -0,0 +1,36 @@
+{{ config(materialized='table') }}
+
+with customers as (
+    select * from {{ ref('stg_customers') }}
+),
+orders as (
+    select * from {{ ref('stg_orders') }}
+),
+payments as (
+		select * from {{ ref('stg_payments') }}
+),
+customer_order_aggregates as (
+    select
+        orders.customer_id,
+        min(orders.order_date) as first_order_date,
+        max(orders.order_date) as most_recent_order_date,
+        count(orders.order_id) as number_of_orders,
+		SUM(payments.amount_australian_dollar) as total_order_amount_australian_dollar
+    from orders
+	LEFT JOIN payments
+	ON payments.order_id = orders.order_id
```

Comment:
This is perfectly fine, but since you already have the join key labeled the same, you can go ahead and use this shorter join condition:
```suggestion
	using (order_id)
```